core/claim.go

```diff
@@ -353,19 +353,21 @@ type ClaimLinkObjectIdentity struct {
 	// IdAddr is the Ethereum Address related to the identity.
 	IdAddr common.Address
 	// ObjectHash is the hash of the object.
-	ObjectHash merkletree.Hash
+	ObjectHash [248 / 8]byte
 }
 
 // NewClaimLinkObjectIdentity returns a ClaimLinkObjectIdentity.
 func NewClaimLinkObjectIdentity(hashType uint32, objectType uint32, objectIndex uint16, idAddr common.Address,
 	objectHash merkletree.Hash) *ClaimLinkObjectIdentity {
```

Comment:
I would suggest using []byte as objectHash type.  It's not a merkletree.Hash because that's a specific hash (mimc7 in this case) which is the hash used internally in the merkle tree.  As it can be different kinds of hash, it may have different lengths.

Reply:
Done! :+1: